src/DayOfWeek.php

```diff
@@ -93,64 +92,63 @@ public static function all(?DayOfWeek $first = null) : array
         do {
             $days[] = $current;
             $current = $current->plus(1);
-        }
-        while (! $current->isEqualTo($first));
+        } while (!$current->isEqualTo($first));
```

Comment:
If possible, I'd like to keep the space here.

Reply:
OK, added `not_operator_with_successor_space` rule